CONTRIBUTING.md

```diff
@@ -52,18 +52,18 @@ and 2017 would be `swb22017`).
 (`Day of Defeat` -> `dod`), then the new name should ignore rule #2 
 (`Day of Dragons` -> `dayofdragons`).
 5. Roman numbering will be converted to arabic numbering (`XIV` -> `14`).
-6. Unless numbers are at the end of a name, they will be considered words, 
-but digits will always be used instead of the acronym (counter to #2) (`Left 4 
-Dead` -> `l4d`) unless they at the start position (`7 Days to Die` -> `sdtd`), 
-if they are at the end (such as sequel number or the year), always append them 
-(`Team Fortress 2` -> `teamfortress2`, `Unreal Tournament 2003` -> 
-`unrealtournament2003`).
+6. Unless numbers (years included) are at the end of a name, they will be considered 
+words. Digits will be used instead of the acronym of that number's digits (`Left 4 
+Dead` -> `l4d`) unless they are at the start position (`7 Days to Die` -> `sdtd`). 
+Other examples: `Team Fortress 2` -> `teamfortress2`, `Unreal Tournament 2003` ->
+`unrealtournament2003`.
```

Comment:
This seems less clear than before. As far as I can tell there's not a defined term for spelling out numbers as words in English [1]. Maybe we could say something like.

```suggestion
6. Unless numbers (years included) are at the end of a name, they will be considered 
words. If a number is not in the first position its entire numeric digits will be used instead of the acronym of that number's digits (`Left 4 
Dead` -> `l4d`). If the number is in the first position the longhand (words: 5 -> five) representation of the number will be used to create an ancronym (`7 Days to Die` -> `sdtd`). 
Other examples: `Team Fortress 2` -> `teamfortress2`, `Unreal Tournament 2003` ->
`unrealtournament2003`.
```


[1]: https://english.stackexchange.com/questions/111170/is-there-a-term-to-denote-writing-numbers-in-words-rather-than-as-digits